fcrepo-search-api/pom.xml

```diff
@@ -47,9 +47,8 @@
 
     <!-- test gear -->
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <scope>test</scope>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
```

Comment:
Unrelated to your work here which is good. But it appears we didn't have any tests but we were importing junit and are still importing mockito. As it doesn't look like you are using it (and it is the old Junit 4 one) maybe we should remove it?